app/view/show.py

```diff
@@ -19,6 +19,9 @@
 
 
 def get_bug_data(cves, pkgs, group):
+
+    MAXLENGTH_SUMMARY = 200
```

Comment:
Ok, should it be as simple as `# Set the max length of the summary as the one defined in bugs.archlinux.org's form` or so?